README.md

```diff
@@ -1,214 +1,174 @@
+check the [installation docs](http://appium.io/docs/en/latest/quickstart/install/) for the
+system requirements and further information.
 
-To install Appium 2 server using Node Package Manager (npm) run the following command:
+If upgrading from Appium 1, make sure Appium 1 is fully uninstalled (`npm uninstall -g appium`).
+Unexpected errors might appear if this has not been done.
 
 ```bash
 npm i -g appium
 ```
 
-To update Appium 2 server using Node Package Manager (npm) run the following command:
-
- ```bash
-npm i -g appium@latest
-```
-
-:bangbang: Make sure Appium 1 is fully uninstalled before upgrading it to Appium 2 (`npm uninstall -g appium`). Unexpected errors might appear if this has not been done.
+Note that this will only install the core Appium server, which cannot automate anything on its own.
+Please install [drivers](#drivers) for your target platforms in order to automate them.
 
-### Drivers
+## Drivers
 
-Appium supports app automation across a variety of platforms, like iOS,
-Android, and Windows. Each platform is supported by one or more "drivers",
-which know how to automate that particular platform. Since version 2.0
-all drivers have been isolated from the Appium server app and can
-be managed independently using the [appium driver](https://appiumpro.com/editions/122-installing-appium-20-and-the-driver-and-plugins-cli) command line interface.
+Appium supports app automation across a variety of platforms, like iOS, Android, macOS, Windows,
+and more. Each platform is supported by one or more "drivers", which know how to automate that
+particular platform. You can find a full list of officially-supported and third-party drivers in
+[Appium Ecosystem's Drivers page](http://appium.io/docs/en/latest/ecosystem/drivers/).
 
-In general, the drivers management in Appium 2 is as simple as:
+Driver management is done using [Appium's Extension command-line interface](http://appium.io/docs/en/latest/cli/extensions/):
 
 ```bash
-# To install a new driver from npm
-appium driver install --source=npm appium-xcuitest-driver[@<version>]
-# To install a driver from a local folder (useful for developers)
-appium driver install --source=local /Users/me/sources/appium-xcuitest-driver
-# To install a new driver from github (hm, maybe it's time to publish it to NPM?)
-appium driver install --source=github --package=appium-xcuitest-driver appium/appium-xcuitest-driver
-
-# To list already installed drivers
+# Install an official driver from npm (see documentation for a list of such drivers)
+appium driver install <driver-name>
+# Install any driver from npm
+appium driver install --source=npm <driver-name>
+# List already installed drivers
 appium driver list --installed
-
-# To update a driver (it must be already installed)
-appium driver update xcuitest
-
-# To uninstall a driver (it won't last forever, wouldn't it?)
-appium driver uninstall xcuitest
+# Update a driver (it must be already installed)
+appium driver update <driver-name>
```

Comment:
I think it also makes sense to add an example with the --unsafe option, so people know how to update to a next major driver version